src/ui/components/SegmentedFontStyleDefinition.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+  import type { UIColor } from '~types/common.ts';
+
+  import { isSupportsOKLCH } from '~ui/constants.ts';
+  import { formatForOklchCSS } from '~utils/colors/formatters.ts';
+  import { isEmpty, notEmpty } from '~utils/not-empty.ts';
+  import { formatHex8 } from 'culori/fn';
+
+  const { currentStyleNumber, id, primaryColor, secondaryColor } = $props<{
```

Comment:
Here is other way to define props types. Let’s use the single one.

Reply:
Ok